src/script/page/RightSidebar/ConversationDetails/ConversationDetails.tsx

```diff
@@ -89,6 +90,8 @@ const ConversationDetails: FC<ConversationDetailsProps> = ({
   isFederated = false,
   isVisible = false,
 }) => {
+  const panelContentRef = useRef<HTMLDivElement>(null);
```

Comment:
you will need a `useEffectRef` here, cause we need a rerender when the ref is updated (ref do not trigger a re-render on their own)